docs/conf.py

```diff
@@ -43,6 +42,8 @@
 
 # -- Options for HTML output -------------------------------------------------
 
+today = str(date.today().strftime('%d %B, %Y'))
```

Comment:
should we pre-defined today to be the last update date for the *CSV file? If the value is not available, then try to use it today (runtime date for this Python script).

Reply:
The date that is currently in place is intended to be the last updated date of the published document (since the csv files in the repo may have been updated since then). Especially the date in the footer, the context is 'this page last updated on ...'. The footer date and the date on the mapping page should match, in order to avoid confusion. Glad to discuss further if needed!